src/types/models/carSales.ts

```diff
@@ -0,0 +1,25 @@
+export interface Buyer {
+  email: string
+  firstName: string
+  lastName: string
+  phone: string
+}
+
+export interface ListingQuery {
```

Comment:
1. Whats your motivation behind the naming?
2. It looks like a subset of the Listing interface instead of redefining the existing props we could use https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys to select the required sumbset